web/components/Home/HowItWorks.js

```diff
@@ -30,25 +30,19 @@ export default function HowItWorks({ flowCharts }) {
       </div>
       <div className="flex items-center space-x-24 select-none">
         {currentChart.steps.map((step, index, steps) => (
-          <>
-            <div
-              className="relative flex flex-col items-center"
-              key={step.title}
-            >
+          <Fragment key={step.title}>
```

Comment:
I am thinking even if the title is unique, but do you think it's a good practice to use it as a key?
maybe the step number or the index

Reply:
fixed